src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/SiriSXUpdaterConfig.java

```diff
@@ -20,7 +20,7 @@ public static SiriSXUpdaterParameters create(String configRef, NodeAdapter c) {
         .since(V2_0)
         .summary("How often the updates should be retrieved.")
         .asDuration(Duration.ofMinutes(1)),
-      c.of("earlyStartSec").since(V2_0).summary("TODO").asInt(-1),
+      c.of("earlyStartSec").since(V2_0).summary("TODO").asInt(0),
```

Comment:
I think this should be a Duration instead.

Also, it would be nice if there was a little bit of documentation.